### PR TITLE
bump to 0.1.7

### PR DESCRIPTION
**Pull request by richardmonette**
Fixes an inadvertent downgrade in activesupport and actionpack verisons, see https://github.com/Shopify/twine/commit/0ce97f679daa9680d96c00227297e5eef167d042#diff-e79a60dc6b85309ae70a6ea8261eaf95R10

@celsodantas @maartenvg @qq99 cc @Shopify/tnt

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -1,49 +1,70 @@
...
 
 GEM
   remote: https://rubygems.org/
   specs:
-    actionpack (4.0.2)
-      activesupport (= 4.0.2)
-      builder (~> 3.1.0)
-      erubis (~> 2.7.0)
-      rack (~> 1.5.2)
+    actionpack (4.2.6)
```
Comment (celsodantas) [inline Gemfile.lock]: :+1:

**Comment by celsodantas** [comment]
slack-imgs com

**Comment by maartenvg** [comment]
This is the diff compared to the previous version ... https://github.com/Shopify/twine/compare/647be0db6a35a20dd7b7eb9ce1f8f0ec809bdb19...0.1.7

File: Gemfile.lock [hunk 2]
```diff
@@ -1,49 +1,70 @@
...
+      rails-deprecated_sanitizer (>= 1.0.1)
+    rails-html-sanitizer (1.0.3)
+      loofah (~> 2.0)
+    railties (4.2.6)
+      actionpack (= 4.2.6)
+      activesupport (= 4.2.6)
       rake (>= 0.8.7)
       thor (>= 0.18.1, < 2.0)
-    rake (10.3.2)
+    rake (11.1.2)
```
Comment (maartenvg) [inline Gemfile.lock]: Is that necessary?

Comment (richardmonette) [inline Gemfile.lock]: No, reverted that change out.

**Pull request merged**